verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -604,5 +608,20 @@ void IndexingFactsTreeExtractor::ExtractPackageImport(
   facts_tree_context_.top().NewChild(package_import_data);
 }
 
+void IndexingFactsTreeExtractor::ExtractQualifiedId(
+    const verible::SyntaxTreeNode& qualified_id) {
+  IndexingNodeData member_reference_data(IndexingFactType::kMemberReference);
+
+  const std::vector<TreeSearchMatch>& unqualified_ids =
```

Comment:
this shouldn't be a reference (it's a dangling reference as FindAllUnqualifiedIds returns a value)